tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "es5",
     "module": "esnext",
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": ["dom", "dom.iterable", "ES2018"],
```

Comment:
Why did we have to make this change?

Reply:
Node v10 implements ES2018 (see [node.green](https://node.green)), so thats our lib level.
This does not affect language features like optional chaining etc, but "stdlib" availability of features like `Object.fromEntries`, which cannot be transpiled by `tsc`.

Reply:
@Skn0tt ok, I think we only need to make this change in the cli package since it's the only one with a plain `tsc` build. All the other packages are transpiled and bundled with rollup.

Reply:
Done in 323e3b3